screens/nadzvychaynaSytuaziya/diyi/diyi.js

```diff
@@ -23,17 +23,19 @@ export default function Diyi({ navigation }) {
           <View style={{ padding: 20 }}>
             <Text style={styles.title}>Надзвичайна ситуація - дії</Text>
 
-            <ListItem
-              roundTop
-              title="Вимкнули телебачення"
-              onPress={()=>navigation.push("Telebachennya")}
-            />
-            <ListItem title="Повітряна тривога" onPress={()=>navigation.push("Povitryana")} />
-            <ListItem title="Евакуація" onPress={navigateAway} />
-            <ListItem title="Вибухонебезпечні знахідки" onPress={()=>navigation.push("VibuhonebezpechniZnahidki")}/>
+            <text style={styles.paragraph}> Нас не готували до війни/кризи. Але ми вміємо просити та приймати 
```

Comment:
The component names are always capital, please rename `text` to be `Text` instead